domain/plan.go

```diff
@@ -153,8 +153,9 @@ func (p *Plan) JobSpecification() JobSpecification {
 
 	for _, task := range p.Tasks {
 		taskInfo := TaskInformation{
-			Id:    task.Id.Hex(),
-			Title: task.Title,
+			Id:       task.Id.Hex(),
```

Comment:
Why do we need these changes? If these changes were for fixing something else, could you perhaps try pulling the latest from the main?

Reply:
Yes I need to pull from the main